### ISSUE TITLE
getAddressTransactions args format

### ISSUE THREAD

**Issue by gagarin55** [ethereumproject/go-ethereum#596]
1. According to spec `QUANTITY` is the integer formatted as hex string. Should we fix it for?
https://github.com/ethereumproject/go-ethereum/wiki/JSON-RPC#geth_getaddresstransactions

2. For parameter 
```
QUANTITY - integer block number to filter transactions ceiling
```

I suggest use type `QUANTITY | TAG`

3. For parameter
```
QUANTITY - integer of index to begin pagination. Using -1 equivalent to 0.
```

I suggest don't introduce -1 value at all. Zero is zero.

**Comment by BelfordZ** [comment]
adding tag -- sounds reasonable. `TAG` here would be the hash of a block to start from?

the `-1` value is a bit odd, but Im guessing its used as a way of saying 'just give me the default' which would be different than saying start from 0, even though they are logically equivalent. I agree that we should just use zero in this case. 

if we are changing this up, I would recommend that we follow `skip` and `limit` as a pattern where `skip` is the number of items to skip, `limit` is the max number of items to return. Currently, the getAddressTransactions api implements the `skip` part pretty much as I'd expect, but `limit` is currently an index to the last item in the set, rather than being a number of results you want.

**Comment by gagarin55** [comment]
`TAG` - the string "latest", "earliest" or "pending"

**Comment by tzdybal** [comment]
It seems that skip/limit aka offset/count is very popular. At least for databases (https://www.postgresql.org/docs/8.1/static/queries-limit.html, https://dev.mysql.com/doc/refman/8.0/en/select.html).

I don't know what's the "usual" approach in APIs.

**Comment by whilei** [comment]
Let's punt the skip/limit stuff to a separate issue and discussion.

**Comment by whilei** [comment]
This was largely a documentation issue -- edits have been made on the wiki corresponding with changes included in v5.4.0: https://github.com/ethereumproject/go-ethereum/wiki/JSON-RPC#geth_getaddresstransactions. The parts of this issue not documentation have been resolved in #598.

If and when the skip/limit topic comes up again that'll be a new thing.
